src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -878,13 +879,11 @@ public static bool TryGetEntityMetadata(CallNode callNode, TexlBinding binding,
 
         // Fetch the description associated with the specified parameter name (which must be the INVARIANT name)
         // If the param has no description, this will return false.
-        public virtual bool TryGetParamDescription(string paramName, out string paramDescription)
+        public virtual bool TryGetParamDescription(string paramName, out string paramDescription, string locale = null)

```

Comment:
>null [](http://example.com/codeflow?start=114&length=4)

Are there existing places that need to pass this locale in?

Reply:
Added to an extra place.